frontend/src/modules/Survey/Styles/StepCourse.style.tsx

```diff
@@ -10,13 +10,21 @@ export const StyledQuestionText = styled.div`
   font-weight: 500;
   line-height: 10px;
   text-align: center;
-  margin-bottom: 120px;
+  margin-bottom: 50px;
 `
 
+export const StyledWarningText = styled.div`
+  ${h4};
+  font-size: 24px;
+  font-weight: 400;
+  text-align: center;
+  margin: 0px 20%;
+  margin-bottom: 50px;
+`
```

Comment:
It would be better to move everything over to `<Typography>` in the future, but this can slide for now